cmd/meroxa/root/apps/deploy.go

```diff
@@ -227,10 +227,11 @@ func (d *Deploy) getPlatformImage(ctx context.Context) (string, error) {
 	if err != nil {
 		return "", err
 	}
-
 	sourceBlob := meroxa.SourceBlob{Url: s.GetUrl}
 	buildInput := &meroxa.CreateBuildInput{SourceBlob: sourceBlob}
-
+	if d.flags.Environment != "" {
+		buildInput.Environment = &meroxa.EntityIdentifier{Name: d.flags.Environment}
```

Comment:
If we're saying we accept either name or UUID, we should check if what was provided via the --env flag was in fact the name or an uuid.

Reply:
How can we check that? Since both are strings passed from flag

Reply:
@anna-cross you can use `uuid.Parse(d.flags.Environment)` and if it errors, then it's a name. Example:

```go
_, err := uuid.Parse(d.flags.Environment)
switch {
case err == nil:
  buildInput.Environment = &meroxa.EntityIdentifier{UUID: d.flags.Environment}
default:
   buildInput.Environment = &meroxa.EntityIdentifier{Name: d.flags.Environment}
}
```